charts/dapr/charts/dapr_sentry/templates/dapr_sentry_deployment.yaml

```diff
@@ -145,7 +145,7 @@ spec:
         - "--log-as-json"
 {{- end }}
 {{- if eq .Values.global.prometheus.enabled true }}
-        - "--enable-metrics"
+        - "--enable-metrics=true"
```

Comment:
This should probably be
```
- "--enable-metrics"
- "true"
```

Reply:
@1046102779 please update as comment said.